### PR TITLE
docs: update the github urls and remove unnecessary mentions

### PR DESCRIPTION
**Pull request by MaferMazu**
## Description

Updates the URLs to use the Open edX ones and removes unnecessary mentions.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @MaferMazu! Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.

File: openedx-commerce.php [hunk 1]
```diff
@@ -1,11 +1,11 @@
 <?php
 /**
  * Plugin Name:       Open edX Commerce
- * Plugin URI:        https://github.com/eduNEXT/openedx-wordpress-ecommerce
+ * Plugin URI:        https://github.com/openedx/openedx-wordpress-ecommerce
  * Description:       Easily connect your WooCommerce store to Open edX.
  * Version:           2.0.3
- * Author:            eduNEXT
- * Author URI:        https://edunext.co/
+ * Author:            The Open edX Community
```
Comment (felipemontoya) [inline openedx-commerce.php:7]: This two I would leave as edunext. I think "the open edx community" is two broad and lacks accountability.

**Review by felipemontoya — COMMENTED** [review]
Removing most mentions to edunext is fine, but I would leave in the author field so that whoever is looking to modify or report bugs or whatnot knows who to expect as the author.

**Review by felipemontoya — APPROVED** [review]
Looks good to me. Thanks for updating all the links

**Comment by openedx-webhooks** [comment]
@MaferMazu 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/openedx-wordpress-ecommerce&entry.867055334=https://github.com/openedx/openedx-wordpress-ecommerce/pull/78&entry.1484655318=https://github.com/MaferMazu&entry.752974735=2024-06-06+13:47&entry.1917517419=2024-06-07+15:31&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**